apps/website/src/app/calendar/calendarStyles.css

```diff
@@ -43,3 +43,38 @@
     margin-top: 1rem;
 }
 
+.calendar-container {
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    width: 100%; /* Ensure the container is full width */
+    padding: 10px; /* Adjust padding as needed */
+    box-sizing: border-box; /* Include padding in the height calculation */
+}
+
+.calendar {
+    flex: 1;
+    overflow: hidden; /* Prevent any overflow from causing scroll */
+    padding: 10px;
+}
+
```

Comment:
no tailwind :(

Reply:
It's an MVP, this can be polished at a later date.